pages/index.vue

```diff
@@ -4,7 +4,7 @@
 
         <!-- Data table file loading area -->
         <b-row>
-            <h2>{{ uiText.dataTableHeader }}</h2>
+            <h3>{{ uiText.dataTableHeader }}</h3>
```

Comment:
Suggestion: I think we can go with smaller headings here. The query tool uses H5.

Reply:
Yeah. I saw that. The issue is that at `h5` the sizing looks a little odd, one of the reasons being because of the large amount of space in the page and another that at `h5` the font size is actually smaller than the app's button font size.